README.md

```diff
@@ -0,0 +1,141 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>My Portfolio</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 My Portfolio ](#-my-portfolio-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 My Portfolio <a name="about-project"></a>
+
+**My Portfolio** is a a website that shows my achievements, my skills, my contact details and all the informations needed by a recruiter or a collaborator
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- Javascript
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Navigation Bar**
+- [ ] **Header**
+- [ ] **My projects**
+- [ ] **About Me**
+- [ ] **Download My Resume**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://Mahamadou-Nouridine.github.io/My-Portfolio/)
+
```

Comment:
- [ ] I think it would be better if you fix the broken `[Live Demo Link](https://Mahamadou-Nouridine.github.io/My-Portfolio/)` live demo link in your README file to make it more professional as it shows a **404** error page once clicked. This occur because you didn't deploy yet your project online. For more information, I suggest you refer this **[link](https://www.youtube.com/watch?v=FJEiNxLxE-E)**
![Screenshot from 2023-03-23 11-45-34](https://user-images.githubusercontent.com/83042742/227258086-202d9102-6f4f-454a-a6ac-4b8bf64d03bc.png)